engine/access/rest/routes/account_keys_test.go

```diff
@@ -0,0 +1,287 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	mocktestify "github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/access/mock"
+	"github.com/onflow/flow-go/model/flow"
+
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// TestGetAccountKeyByIndex tests local getAccountKeyByIndex request.
+//
+// Runs the following tests:
+// 1. Get key by address and index at latest sealed block.
+// 2. Get key by address and index at latest finalized block.
+// 3. Get missing key by address and index at latest sealed block.
+// 4. Get missing key by address and index at latest finalized block.
+// 5. Get key by missing address and index at latest sealed block.
+// 6. Get key by missing address and index at latest finalized block.
+// 7. Get key by address and index at height.
+func TestGetAccountKeyByIndex(t *testing.T) {
+	backend := &mock.API{}
```

Comment:
this will automatically assert expectation
```suggestion
	backend := mock.NewAPI()
```

Reply:
Fixed in https://github.com/onflow/flow-go/pull/4583/commits/25374df29468ae28971873743a216139be57b814